lib/ansible/plugins/inventory/cloudscale.py

```diff
@@ -0,0 +1,184 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright: (c) 2018, Gaudenz Steinlin <gaudenz.steinlin@cloudscale.ch>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+name: cloudscale
+plugin_type: inventory
+author:
+  - Gaudenz Steinlin (@gaudenz)
+  - René Moser (@resmo)
```

Comment:
it is all you...

Reply:
I thouht this is required for you to be recognized by ansibot as an owner of the module. If you want me to remove your name I can do that, but I have no problem with you being listed here.

Reply:
it is not required for the bot, there is a separate config for the bot handling. Thanks for the credit, but I didn't write a single line. Please remove me from the author list.